tap_bing_ads/__init__.py

```diff
@@ -54,6 +58,40 @@
 
 ARRAY_TYPE_REGEX = r'ArrayOf([A-Za-z0-9]+)'
 
+def should_retry_httperror(exception):
+    """ Return true if exception is required to retry otherwise return false """
+    try:
+        if isinstance(exception, ConnectionError) or isinstance(exception, ssl.SSLError):
+            return True
+        elif isinstance(exception, suds.transport.TransportError) or isinstance(exception, socket.timeout):
```

Comment:
Why can't we merge the `elif` statements in the `if` statement?

Reply:
Merged `elif` statements in the `if` statement condition except condition for error code of 408 because of better readability.